admin-dev/themes/default/scss/partials/_nav.scss

```
@@ -485,7 +497,7 @@
   }
 
   .material-icons {
-    color: #bebebe;
+    color: #fff;
```

```suggestion
    color: $gray-dark-link;
```

I think we want this one to be the same color as links no?

---

Anyway, we need to use an SCSS variable